src/node/traceur.js

```diff
@@ -28,5 +28,8 @@ module.exports = {
   __proto__: traceur,
   get require() {
     return require('./require.js');
+  },
+  get binFilename() {
```

Comment:
binaryFilename seems better after all... compile* seems to imply something about the target output file.